frontend_lib/test/fixture/tracimLiveMessage/commentTlm.js

```diff
@@ -1,9 +1,10 @@
 import { author } from '../author.js'
+import { CONTENT_TYPE } from '../../../src'
```

Comment:
Is this import right? Maybe it's:
 ```suggestion
import { CONTENT_TYPE } from '../../../src/helper.js'
```